### ISSUE TITLE
validate_vars aborts if optional "github_token" not set

### ISSUE THREAD

**Issue by mbarnes** [jlebon/redhat-ci#55]
Was messing around with running `papr` locally and hit this right off the bat:
```
ERROR: Missing variable 'github_token'.
```
The `github_token` variable is supposed to be optional, according to [RUNNING.md](https://github.com/projectatomic/papr/blob/master/docs/RUNNING.md).

My workaround was just to disable the `validate_vars` call in `main`, or I guess remove `github_token` from the loop leaving only `github_repo` to validate.

**Comment by jlebon** [comment]
Yeah, that sounds about right. We regressed on this at some point. Though note the whole codebase is in flux while we move to Python, so some of the documentation might be off until things settle.
